packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -20,6 +21,12 @@ export type GraphQLPersonalizeServiceConfig = {
    * Timeout (ms) for the Personalize request. Default is 250.
    */
   timeout?: number;
+  /**
+   * Cache settings
+   * Specified the cache timeout and whether cache is enabled
+   * If not provided cache will be enabled with timeout of 10 seconds
+   */
+  cacheSettings?: CacheOptions;
```

Comment:
Should we simply extend `GraphQLPersonalizeServiceConfig` instead? e.g. `export type GraphQLPersonalizeServiceConfig = CacheOptions & {...}`. This would be more consistent with dictionary service (where cache props are at the root of the config).

Reply:
I think it's better to have cache options as its own field, for clarity.
Maybe we can revisit this when streamlining all gql services in the future?

Reply:
If the props weren't prefixed with "cache", I'd agree - but since they are it actually is a bit repetitive/redundant. e.g. 
`new GraphQLPersonalizeService({ cacheSettings: { cacheTimeout: 5 }, endpoint, ... }` vs 
`new GraphQLPersonalizeService({ cacheTimeout: 5, endpoint, ... }`

Reply:
Fair enough. Changed